cmake/Futility_Configurations.cmake

```diff
@@ -37,6 +37,7 @@ ENDIF()
 SET(CONFIG_TYPES
     DEBUG
     RELEASE
+    RELWITHDEBINFO
```

Comment:
can we just follow CASL RELEASE_DEBUG syntax so I don't have to remember 2 crazy names?

Reply:
Tribit specs

Reply:
I'll allow it...